app/controllers/category_transactions_controller.rb

```diff
@@ -0,0 +1,39 @@
+class CategoryTransactionsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :find_category, only: :new
```

Comment:
@sinpai why you use `#find_category` before `#new`, and don't use `@category` in `new` view?

Reply:
Thanks, it's useless.